### DISCUSSION TITLE
[Feature request] Add speed selector for Replay Live Timing

### DISCUSSION THREAD

**Discussion by liuxuan30** [f1multiviewer/issue-tracker#569] [Feature requests]
Hi there,

Nice app! I was playing Qatar GP main race at 1.25x speed, and I notice the live timing data board playback seems still at normal speed, not synced to 1.25x speed, nor place or options to change the live timing speed. Sounds like a bug to me.

**Comment by LapsTimeOFF** [comment]
Hey, this is an expected behaviour as of right now, I will convert this to a feature request.

**Comment by LapsTimeOFF** [comment]
Duplicate of #519
